rs/rosetta-api/ledger_core/src/balances.rs

```diff
@@ -34,8 +34,8 @@ where
     type AccountId = AccountId;
     type Tokens = Tokens;
 
-    fn get_balance(&self, k: &Self::AccountId) -> Option<&Self::Tokens> {
-        self.get(k)
+    fn get_balance(&self, k: &Self::AccountId) -> Option<Self::Tokens> {
+        self.get(k).cloned()
```

Comment:
In `get_balances()` of `impl BalancesStore for ClientBalancesStore`, you call `copied()`, whereas here you call `cloned()`. In this particular case I'm not sure it makes a big difference, but for consistency I would use the same approach in both cases, and here I would probably go for `copied()`, as long as all the fields in `Tokens` also implement/derive the `Copy` trait, as is currently the case (for the single `e8s` field).

Reply:
Since `TokensType` does not implement `Copy`, I changed both to `cloned()`, PTAL!